openslides/agenda/static/js/agenda/site.js

```diff
@@ -268,13 +269,22 @@ angular.module('OpenSlidesApp.agenda.site', [
                     return false;
             }
         };
-        $scope.getUpdateStatePrefix = function (item) {
+        $scope.getDetailStatePrefix = function (item) {
             var prefix = item.content_object.collection.replace('/','.');
             // Hotfix for Issue 2566.
             // The changes could be reverted if Issue 2480 is closed.
             prefix = prefix.replace('motion-block', 'motionBlock');
             return prefix;
         };
+        $scope.edit = function (item) {
+            var formName = item.content_object.collection.split('/')[1];
+            // Hotfix for Issue 2566.
+            // The changes could be reverted if Issue 2480 is closed.
+            formName = formName.replace('motion-block', 'motionBlock');
+            formName = formName.charAt(0).toUpperCase() + formName.slice(1) + 'Form';
```

Comment:
Hm. This does not work if I have a collection foo/bar-baz. It would be transformed to Bar-bazForm instead of BarBazForm.